script/upgrades/MU06/MU06.sol

```diff
@@ -0,0 +1,338 @@
+import { IPricingModule } from "mento-core-2.4.0/interfaces/IPricingModule.sol";
+import { IReserve } from "mento-core-2.4.0/interfaces/IReserve.sol";
+
+import { Broker } from "mento-core-2.4.0/swap/Broker.sol";
+import { TradingLimits } from "mento-core-2.4.0/libraries/TradingLimits.sol";
+import { BreakerBox } from "mento-core-2.4.0/oracles/BreakerBox.sol";
+import { ValueDeltaBreaker } from "mento-core-2.4.0/oracles/breakers/ValueDeltaBreaker.sol";
+import { Reserve } from "mento-core-2.4.0/swap/Reserve.sol";
+
+import { MU06Config, Config } from "./Config.sol";
+import { IMentoUpgrade, ICeloGovernance } from "script/interfaces/IMentoUpgrade.sol";
+
+/**
+ forge script {file} --rpc-url $BAKLAVA_RPC_URL 
+                     --broadcast --legacy 
+ * @dev depends on: ../deploy/*.sol
+ */
+contract MU06 is IMentoUpgrade, GovernanceScript {
+  using TradingLimits for TradingLimits.Config;
+  using FixidityLib for FixidityLib.Fraction;
+
+  ICeloGovernance.Transaction[] private transactions;
+
+  // Tokens
+  address private cUSD;
+  address private nativeUSDT;
+  address private nativeUSDC;
+  address private bridgedUSDC;
+
+  // Mento contracts
+  address private breakerBox;
+  address private valueDeltaBreaker;
+  address private brokerProxy;
+  address private biPoolManagerProxy;
+  address payable private reserveProxy;
+
+  bool public hasChecks = true;
+
+  function prepare() public {
+    loadDeployedContracts();
+    setAddresses();
+  }
+
+  /**
+   * @dev Loads the deployed contracts from previous deployments
+   */
+  function loadDeployedContracts() public {
+    contracts.load("MU01-00-Create-Proxies", "latest"); // BrokerProxy & BiPoolProxy
+    contracts.load("MU01-01-Create-Nonupgradeable-Contracts", "latest"); // Value Delta Breaker
+    contracts.load("MU03-01-Create-Nonupgradeable-Contracts", "latest"); // BreakerBox & ConstantSumPricingModule
+  }
+
+  /**
+   * @dev Sets the addresses of the various contracts needed for the proposal.
+   */
+  function setAddresses() public {
+    // Tokens
+    cUSD = contracts.celoRegistry("StableToken");
+    nativeUSDT = contracts.dependency("NativeUSDT");
+    nativeUSDC = contracts.dependency("NativeUSDC");
+    bridgedUSDC = contracts.dependency("BridgedUSDC");
+
+    // Oracles
+    breakerBox = contracts.deployed("BreakerBox");
+    valueDeltaBreaker = contracts.deployed("ValueDeltaBreaker");
+
+    // Swaps
+    brokerProxy = contracts.deployed("BrokerProxy");
+    biPoolManagerProxy = contracts.deployed("BiPoolManagerProxy");
+    reserveProxy = address(uint160(contracts.celoRegistry("Reserve")));
+  }
+
+  function run() public {
+    prepare();
+
+    address governance = contracts.celoRegistry("Governance");
+    ICeloGovernance.Transaction[] memory _transactions = buildProposal();
+
+    vm.startBroadcast(Chain.deployerPrivateKey());
+    {
+      createProposal(_transactions, "SET ME PLS :'(", governance);
```

Comment:
```suggestion
      createProposal(_transactions, "https://forum.celo.org/t/proposal-mento-upgrade-mu06-native-usdt-integration/8010", governance);
```

Reply:
This link doesn't work, it has to be the MD file from the CGP. I will revert if so that we don't end up using it by mistake